recipes/autograd/meta.yaml

```diff
@@ -0,0 +1,75 @@
+package:
+  name: autograd
+  version: "1.1.5"
+
+source:
+  fn: autograd-1.1.5.tar.gz
+  url: https://files.pythonhosted.org/packages/e5/70/e9c8db296d6ff33a17fec0ecaf11e3efbc85722d042365d01704046befef/autograd-1.1.5.tar.gz
+  md5: 2b65978109361a6ffb6986d03aebc157
+#  patches:
+   # List any patch files here
+   # - fix.patch
+
+build:
+  # noarch_python: True
+  # preserve_egg_dir: True
+  # entry_points:
+    # Put any entry points (scripts to be generated automatically) here. The
+    # syntax is module:function.  For example
+    #
+    # - autograd = autograd:main
+    #
+    # Would create an entry point called autograd that calls autograd.main()
+
+
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  number: 0
```

Comment:
You can replace the two build scripts with `script: python setup.py install --single-version-externally-managed --record=record.txt`